src/en/index.html

```diff
@@ -22,30 +22,41 @@ <h2 class="h1 mb-8">The Future of Storage<sup>™</sup></h2>
 <section class="max-w-192 mx-auto section text-center">
   <h2 class="h2">Reliable and scalable storage designed for any organization</h2>
   <p class="mb-0 standout">
-    Use Ceph to transform your storage infrastructure. Ceph provides a unified storage service with object, block, and file interfaces from
-    a single cluster built from commodity hardware components. Whether your workload runs on-prem, in the public or private cloud, or is
-    container native, Ceph is applicable to wide range of use cases.
+    Use Ceph to transform your storage infrastructure. Ceph provides a unified
+    storage service with object, block, and file interfaces from a single
+    cluster built from commodity hardware components.
   </p>
 </section>
 
 <section class="bg-grey-300 section section--full">
   <div class="wrapper">
+	  <div>
+        <h2 class="h2">Deploy or manage a Ceph cluster</h2>
+        <p class="p mb-8">deploy ceph now. use the links below to acquire ceph and deploy a ceph cluster.</p>
```

Comment:
does capitalization matter here or does the style take care of it?

Reply:
Captilization here matters, yep

Reply:
I'll resolve this when I update the layout.

Reply:
Wow, that's weird. I suspect that I pressed "gu" after selecting all the text in the tags with "vit" in vim. If that's all Darmok and Jilad at Tenagra to you, just know that this is my fault and that I caused it by using vim.